modules/api-svc/src/OutboundServer/handlers.js

```diff
@@ -310,7 +310,7 @@ const putBulkTransactions = async (ctx) => {
             ctx.state.eventLogger.info(`Sent domain event ${msg.getName()}`);
         }
 
-        ctx.response.status = 204;
+        ctx.response.status = 202;
```

Comment:
Can we use this enum for HTTP Status codes from central-services-shared? --> https://github.com/mojaloop/central-services-shared/blob/470f6e9a580b28d8def9d19519593d14bb990ecf/src/enums/http.js#L145